[2025-03-15T06:20:44.641+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: gen_data_daily.get_date scheduled__2025-03-13T22:00:00+00:00 [queued]>
[2025-03-15T06:20:44.656+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: gen_data_daily.get_date scheduled__2025-03-13T22:00:00+00:00 [queued]>
[2025-03-15T06:20:44.657+0000] {taskinstance.py:2171} INFO - Starting attempt 1 of 2
[2025-03-15T06:20:44.679+0000] {taskinstance.py:2192} INFO - Executing <Task(PythonOperator): get_date> on 2025-03-13 22:00:00+00:00
[2025-03-15T06:20:44.688+0000] {standard_task_runner.py:60} INFO - Started process 987 to run task
[2025-03-15T06:20:44.696+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'gen_data_daily', 'get_date', 'scheduled__2025-03-13T22:00:00+00:00', '--job-id', '8', '--raw', '--subdir', 'DAGS_FOLDER/gen_data_daily.py', '--cfg-path', '/tmp/tmp64x3cin8']
[2025-03-15T06:20:44.698+0000] {standard_task_runner.py:88} INFO - Job 8: Subtask get_date
[2025-03-15T06:20:44.781+0000] {task_command.py:423} INFO - Running <TaskInstance: gen_data_daily.get_date scheduled__2025-03-13T22:00:00+00:00 [running]> on host e9f75e17acd8
[2025-03-15T06:20:45.267+0000] {taskinstance.py:2481} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='longnv' AIRFLOW_CTX_DAG_ID='gen_data_daily' AIRFLOW_CTX_TASK_ID='get_date' AIRFLOW_CTX_EXECUTION_DATE='2025-03-13T22:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-13T22:00:00+00:00'
[2025-03-15T06:20:45.278+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/context.py:315: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(k, replacements))

[2025-03-15T06:20:45.280+0000] {python.py:201} INFO - Done. Returned value was: 2025-03-14
[2025-03-15T06:20:45.327+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=gen_data_daily, task_id=get_date, execution_date=20250313T220000, start_date=20250315T062044, end_date=20250315T062045
[2025-03-15T06:20:45.376+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-03-15T06:20:45.422+0000] {taskinstance.py:3281} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-03-15T15:59:34.744+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: gen_data_daily.get_date scheduled__2025-03-13T22:00:00+00:00 [queued]>
[2025-03-15T15:59:34.763+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: gen_data_daily.get_date scheduled__2025-03-13T22:00:00+00:00 [queued]>
[2025-03-15T15:59:34.763+0000] {taskinstance.py:2171} INFO - Starting attempt 1 of 2
[2025-03-15T15:59:34.786+0000] {taskinstance.py:2192} INFO - Executing <Task(PythonOperator): get_date> on 2025-03-13 22:00:00+00:00
[2025-03-15T15:59:34.794+0000] {standard_task_runner.py:60} INFO - Started process 71 to run task
[2025-03-15T15:59:34.799+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'gen_data_daily', 'get_date', 'scheduled__2025-03-13T22:00:00+00:00', '--job-id', '3', '--raw', '--subdir', 'DAGS_FOLDER/gen_data_daily.py', '--cfg-path', '/tmp/tmpd0xoupip']
[2025-03-15T15:59:34.802+0000] {standard_task_runner.py:88} INFO - Job 3: Subtask get_date
[2025-03-15T15:59:34.904+0000] {task_command.py:423} INFO - Running <TaskInstance: gen_data_daily.get_date scheduled__2025-03-13T22:00:00+00:00 [running]> on host d80566e4f2e2
[2025-03-15T15:59:35.030+0000] {taskinstance.py:2481} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='longnv' AIRFLOW_CTX_DAG_ID='gen_data_daily' AIRFLOW_CTX_TASK_ID='get_date' AIRFLOW_CTX_EXECUTION_DATE='2025-03-13T22:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-13T22:00:00+00:00'
[2025-03-15T15:59:35.048+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/context.py:315: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(k, replacements))

[2025-03-15T15:59:35.048+0000] {python.py:201} INFO - Done. Returned value was: 2025-03-14
[2025-03-15T15:59:35.100+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=gen_data_daily, task_id=get_date, execution_date=20250313T220000, start_date=20250315T155934, end_date=20250315T155935
[2025-03-15T15:59:35.141+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-03-15T15:59:35.184+0000] {taskinstance.py:3281} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-03-15T19:38:37.450+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: gen_data_daily.get_date scheduled__2025-03-13T22:00:00+00:00 [queued]>
[2025-03-15T19:38:37.493+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: gen_data_daily.get_date scheduled__2025-03-13T22:00:00+00:00 [queued]>
[2025-03-15T19:38:37.494+0000] {taskinstance.py:2171} INFO - Starting attempt 1 of 2
[2025-03-15T19:38:37.512+0000] {taskinstance.py:2192} INFO - Executing <Task(PythonOperator): get_date> on 2025-03-13 22:00:00+00:00
[2025-03-15T19:38:37.524+0000] {standard_task_runner.py:60} INFO - Started process 2043 to run task
[2025-03-15T19:38:37.529+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'gen_data_daily', 'get_date', 'scheduled__2025-03-13T22:00:00+00:00', '--job-id', '30', '--raw', '--subdir', 'DAGS_FOLDER/gen_data_daily.py', '--cfg-path', '/tmp/tmpc5vf9q9u']
[2025-03-15T19:38:37.531+0000] {standard_task_runner.py:88} INFO - Job 30: Subtask get_date
[2025-03-15T19:38:37.605+0000] {task_command.py:423} INFO - Running <TaskInstance: gen_data_daily.get_date scheduled__2025-03-13T22:00:00+00:00 [running]> on host 42e0710fef4e
[2025-03-15T19:38:37.712+0000] {taskinstance.py:2481} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='longnv' AIRFLOW_CTX_DAG_ID='gen_data_daily' AIRFLOW_CTX_TASK_ID='get_date' AIRFLOW_CTX_EXECUTION_DATE='2025-03-13T22:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-13T22:00:00+00:00'
[2025-03-15T19:38:37.714+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/context.py:315: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(k, replacements))

[2025-03-15T19:38:37.714+0000] {python.py:201} INFO - Done. Returned value was: 2025-03-14
[2025-03-15T19:38:37.750+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=gen_data_daily, task_id=get_date, execution_date=20250313T220000, start_date=20250315T193837, end_date=20250315T193837
[2025-03-15T19:38:37.790+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-03-15T19:38:37.820+0000] {taskinstance.py:3281} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-03-17T17:47:38.446+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: gen_data_daily.get_date scheduled__2025-03-13T22:00:00+00:00 [queued]>
[2025-03-17T17:47:38.477+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: gen_data_daily.get_date scheduled__2025-03-13T22:00:00+00:00 [queued]>
[2025-03-17T17:47:38.477+0000] {taskinstance.py:2171} INFO - Starting attempt 1 of 2
[2025-03-17T17:47:38.489+0000] {taskinstance.py:2192} INFO - Executing <Task(PythonOperator): get_date> on 2025-03-13 22:00:00+00:00
[2025-03-17T17:47:38.499+0000] {standard_task_runner.py:60} INFO - Started process 530 to run task
[2025-03-17T17:47:38.503+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'gen_data_daily', 'get_date', 'scheduled__2025-03-13T22:00:00+00:00', '--job-id', '15', '--raw', '--subdir', 'DAGS_FOLDER/gen_data_daily.py', '--cfg-path', '/tmp/tmp4xythbya']
[2025-03-17T17:47:38.505+0000] {standard_task_runner.py:88} INFO - Job 15: Subtask get_date
[2025-03-17T17:47:38.585+0000] {task_command.py:423} INFO - Running <TaskInstance: gen_data_daily.get_date scheduled__2025-03-13T22:00:00+00:00 [running]> on host d107d9d70093
[2025-03-17T17:47:38.699+0000] {taskinstance.py:2481} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='longnv' AIRFLOW_CTX_DAG_ID='gen_data_daily' AIRFLOW_CTX_TASK_ID='get_date' AIRFLOW_CTX_EXECUTION_DATE='2025-03-13T22:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-13T22:00:00+00:00'
[2025-03-17T17:47:38.701+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/context.py:315: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(k, replacements))

[2025-03-17T17:47:38.702+0000] {python.py:201} INFO - Done. Returned value was: 2025-03-14
[2025-03-17T17:47:38.747+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=gen_data_daily, task_id=get_date, execution_date=20250313T220000, start_date=20250317T174738, end_date=20250317T174738
[2025-03-17T17:47:38.808+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-03-17T17:47:38.842+0000] {taskinstance.py:3281} INFO - 1 downstream tasks scheduled from follow-on schedule check
